src/effects/VST/VSTEffect.cpp

```diff
@@ -952,7 +950,8 @@ std::shared_ptr<EffectInstance> VSTEffect::DoMakeInstance()
    GetConfig(*this, PluginSettings::Shared, wxT("Options"),
       wxT("UseLatency"), mUseLatency, true);
    mBlockSize = mUserBlockSize;
-   return std::make_shared<Instance>(*this);
+ 
+   return std::make_shared<VSTEffectInstance>(*this, mPath);
```

Comment:
An earlier version of this PR copied mUseLatency into the Instance in its constructor.  That change should remain.

What I should have said too:  the block size, and the user block size, should also be passed into the constructor of the instance.

Also the array of slaves used to be passed as a reference into the constructor.  Why no longer?  Do that too.

Reply:
mUseLatency, block size, user block size are now passed into the constructor of the instance and saved there.

I can not remember ever passing the array of slaves to the instance? and I wonder why I should do it now. I converted slaves to be instances. Why would a slave/instance need to know the other slaves/instances?